config/v1/types_infrastructure.go

```diff
@@ -536,6 +536,14 @@ type AWSPlatformStatus struct {
 	// +optional
 	// +nullable
 	CloudLoadBalancerConfig *CloudLoadBalancerConfig `json:"cloudLoadBalancerConfig,omitempty"`
+
+	// ipFamily indicates the IP Address family supported by cluster nodes.
+	// +default="IPFamiliesIPv4"
+	// +kubebuilder:default:="IPFamiliesIPv4"
```

Comment:
Are we sure we want to default this? Are there existing clusters where this could default to an incorrect value?

Reply:
We are adding an install-config field for customers to set the IPFamily for the cluster for new installs. There too we are defaulting to IPV4. And as part of the support for dualstack, the Installer will always be explicitly setting this field in the Infra CR when the manifest is generated.

For existing clusters, we know that they are all IPv4 and hence added this defaulting behavior. Its value should be immutable.